sources/assets/Stride.Core.Assets/PackageSessionPublicHelper.cs

```diff
@@ -16,6 +16,9 @@ namespace Stride.Core.Assets
     /// </summary>
     public static class PackageSessionPublicHelper
     {
+
+        public const byte NetMajorVersion = 8; //or NET_MAJOR_VERSION
```

Comment:
@Doprez This comment was only to suggest the alternative style of variable naming. Please, delete the comment 😅

Reply:
oop done!